packages/react/src/control-group/ControlGroup.tsx

```diff
@@ -49,10 +50,7 @@ export const ControlGroup = ({
 			invalid={invalid}
 		>
 			<FieldContainer invalid={invalid} id={groupId}>
-				<fieldset
-					aria-describedby={describedBy}
-					css={{ padding: 0, margin: 0, border: 'none' }}
-				>
+				<Box as="fieldset" aria-describedby={describedBy}>
```

Comment:
Changing all `<fieldset>` to `<Box as="fieldset">` so we're not creating additional classes for things that just need resetting